app/models/template.rb

```diff
@@ -25,6 +26,8 @@ class Template < ApplicationRecord
 
   attr_exportable :name, :description, :snippet, :template_inputs, :model => ->(template) { template.class.to_s }
 
+  scoped_search relation: :cloned_from, on: :cloned_from_id, complete_value: false
```

Comment:
Why do we need to search by template id? I am not even sure the id is visible in the UI.

Reply:
I guess searching by name would be more user-friendly, but I'm unsure if it is possible with the `scoped_search` gem.

Reply:
If you can't find a proper way in the [documentation](https://github.com/wvanbergen/scoped_search/wiki/search-definition), you can always use the [`ext_method`](https://github.com/wvanbergen/scoped_search/wiki/search-definition#external-search-methods), like we did [here](https://github.com/theforeman/foreman/blob/8283db898b7e145e59dd376ac0b4cb2d30f7b436/app/models/fact_value.rb#L20)

Reply:
Hmm let's remove it for now, I'd like to keep the PR as small as possible